internal/services/hdinsight/hdinsight_interactive_query_cluster_resource.go

```diff
@@ -32,7 +32,7 @@ var hdInsightInteractiveQueryClusterWorkerNodeDefinition = HDInsightNodeDefiniti
 	CanSpecifyInstanceCount: true,
 	MinInstanceCount:        1,
 	CanSpecifyDisks:         false,
-	CanAutoScaleByCapacity:  true,
+	CanAutoScaleByCapacity:  false,
```

Comment:
This is a breaking change. This used to work which means some people may still have configuration built on it. We should deprecate this instead

Reply:
Updated to deprecate in 4.0.